src/styles/main.scss

```diff
@@ -0,0 +1,68 @@
+$stop-watch-size: 600px;
+$rod-size: 20px;
+$watch-rod-circle: 50%;
+$hands-indent: 50%;
+
+@mixin position-center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+}
+
+@mixin position-transform-hands {
+  position: absolute;
+  left: $hands-indent;
+  transform-origin: bottom;
+}
+
+body {
+  margin: 0;
+}
+
+.stop-watch {
+  @include position-center;
+  width: $stop-watch-size;
+  height: $stop-watch-size;
+  background-color: #d0bce7eb;
+  border-radius: $watch-rod-circle;
+  transform: translate(-$watch-rod-circle, -$watch-rod-circle);
+
+  &__rod {
+    @include position-center;
+    height: $rod-size;
+    width: $rod-size;
+    background-color: #0120b9;
+    border-radius: $watch-rod-circle;
+    transform: translate(-$watch-rod-circle, -$watch-rod-circle);
+    z-index: 1;
+  }
+
+  &__minute {
+    @include position-transform-hands;
+    height: 300px;
+    width: 3px;
+    background-color: #3cda0c;
+    border-radius: 3px;
+    animation: rotate 3600s steps(60) infinite;
+  }
+
+  &__second {
+    @include position-transform-hands;
+    top: -8%;
+    height: 350px;
```

Comment:
a bit strange that the arrow is longer than the stopwatch radius:
![image](https://user-images.githubusercontent.com/86923089/221404036-43f56230-d56e-40dc-a971-280ca43cf657.png)